cryptographic_estimators/SDEstimator/SDAlgorithms/bjmm_dw.py

```diff
@@ -19,56 +19,56 @@
 from ...base_algorithm import optimal_parameter
 from ...SDEstimator.sd_algorithm import SDAlgorithm
 from ...SDEstimator.sd_problem import SDProblem
-from ...SDEstimator.sd_helper import _gaussian_elimination_complexity, _mem_matrix, _mitm_nn_complexity, binom, log2, \
-    ceil, inf
+from ...SDEstimator.sd_helper import (
+    _gaussian_elimination_complexity,
+    _mem_matrix,
+    _mitm_nn_complexity,
+    binom,
+    log2,
+    ceil,
+    inf,
+)
 from scipy.special import binom as binom_sp
 from scipy.optimize import fsolve
 from warnings import filterwarnings
 from types import SimpleNamespace
 from ..sd_constants import *
+
 filterwarnings("ignore", category=RuntimeWarning)
 
 
 class BJMMdw(SDAlgorithm):
     def __init__(self, problem: SDProblem, **kwargs):
-        """
-        Construct an instance of BJMM's estimator using *d*isjoint *w*eight distributions combined with
-         MitM-nearest neighbor search. [EB22]_, [MMT11]_, [BJMM12]_.
+        """Construct an instance of BJMM's estimator using disjoint weight distributions combined with MitM-nearest neighbor search. [EB22]_, [MMT11]_, [BJMM12]_.
```

Comment:
Here the underscores were not removed. @Dioprz  we have to be careful with this cases when adding the `_` back to those for which it was removed